check_install.py

```diff
@@ -11,6 +11,7 @@ def main():
     parser = argparse.ArgumentParser(description='test_login cmdline parser')
     parser.add_argument('-u', default="admin", help='login user name')
     parser.add_argument('-P', default="YourPaSsWoRd", help='login password')
+    parser.add_argument('-P2', default="Test@2022", help='new password')
```

Comment:
This will conflict with `-P`. For multiple char option, suggest `--P2`.
`-P2` == `-P` + `-2` in many command option conventions.
ref: https://stackoverflow.com/a/21286343

Reply:
This is still applicable.

Reply:
Agree, can it be some literal line '-N', default="Test@2022", help='new password'  ?